### PR TITLE
Renaming to labs-post

### PR DESCRIPTION
**Pull request by giorgiosironi**

### CODE REVIEW

File: src/api.raml [hunk 1]
```diff
@@ -791,19 +791,19 @@ traits:
                 description: |
                     List of experiments, sorted by number.
```
Comment (thewilkybarkid) [inline src/api.raml]: This (and above) needs changing. We should sort like blog articles (ie date).

File: src/api.raml [hunk 2]
```diff
@@ -777,7 +777,7 @@ traits:
                                     displayName: Complete
                                     value: !include samples/interview/v1/complete.json
 
-/labs-experiments:
+/labs-posts:
     type: collection
     description: |
         Labs experiments
```
Comment (thewilkybarkid) [inline src/api.raml]: Not anymore.

File: src/model/search.v1.yaml [hunk 3]
```diff
@@ -134,7 +134,7 @@ properties:
                 description: Number of interviews in the result set
                 type: integer
                 minimum: 0
-            labs-experiment:
+            labs-post:
                 description: Number of Labs experiments in the result set
```
Comment (thewilkybarkid) [inline src/model/search.v1.yaml]: More

File: src/samples/search/v1/first-page.json [hunk 4]
```diff
@@ -71,7 +71,7 @@
             ]
         },
         {
-            "type": "labs-experiment",
+            "type": "labs-post",
             "number": 1,
```
Comment (thewilkybarkid) [inline src/samples/search/v1/first-page.json]: Should be an `id`.

File: src/snippets/labs-post.v1.yaml [hunk 5]
```diff
@@ -2,9 +2,9 @@ $schema: http://json-schema.org/draft-04/schema#
 title: Labs experiment snippet
 type: object
 properties:
-    number:
-        description: Number
-        type: integer
+    id:
```
Comment (thewilkybarkid) [inline src/snippets/labs-post.v1.yaml]: `required` needs updating.

File: src/snippets/labs-post.v1.yaml [hunk 6]
```diff
@@ -2,9 +2,9 @@ $schema: http://json-schema.org/draft-04/schema#
...
+        description: Identifier
```
Comment (thewilkybarkid) [inline src/snippets/labs-post.v1.yaml]: This should be like the others and be `$ref: ../misc/id.v1.yaml`

**Comment by nlisgo** [comment]
Can this be rebased and merged in?

**Comment by thewilkybarkid** [comment]
Is Journal CMS just about ready?

**Comment by nlisgo** [comment]
No, I just normally would pull in the latest `elife/api` code while doing the journal-cms refactor. If you prefer for me to do the work first then I will.

**Comment by thewilkybarkid** [comment]
It's quite a disruptive change so we won't be able to merge anything in the meantime (not aware of anything needed, but...). If you could get as far as you can without then that would be appreciated.

**Comment by nlisgo** [comment]
Sure.

File: src/api.raml [hunk 7]
```diff
@@ -788,63 +788,63 @@ traits:
...
-                                value: !include samples/labs-experiment-list/v1/first-page.json
+                                value: !include samples/labs-post-list/v1/first-page.json
                             empty:
                                 displayName: Empty list
                                 description: |
                                     Demonstration of the list being empty (the first page will always return a valid result).
-                                value: !include samples/labs-experiment-list/v1/empty.json
+                                value: !include samples/labs-post-list/v1/empty.json
 
     /{number}:
```
Comment (nlisgo) [inline src/api.raml]: I believe this should be `{id}`

File: src/api.raml [hunk 8]
```diff
@@ -788,63 +788,63 @@ traits:
...
-                resource: Experiment
+                resource: Post
         description: |
-            Individual Labs experiment.
+            Individual Labs post.
         uriParameters:
             number:
                 description: |
-                    Experiment number.
+                    Post number.
```
Comment (thewilkybarkid) [inline src/api.raml]: ID

**Pull request merged**